### PR TITLE
[Kommander] MinIO attributions in download locations

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-89883

## Description of changes being made

Added minIO attributions to Kommander versions 2.3, 2.2, 2.1, and 1.4 in locations where the product is downloaded.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4483.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by mwrightd2** [comment]
Hey all, this PR can also be edited to add updated .tar names once we've settled on those. Just putting this up for the time being.

File: pages/dkp/kommander/1.4/install-airgapped/index.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ Your `cluster.yaml` file should look similar to the following for Kommander Addo
 
  If you wish to disable the Docker registry certificate verification, set `docker-registry-insecure-skip-tls-verify` to `true` in the `utilityApiserver`'s `extraArgs`. We encourage you to keep the certificate verification enabled to validate all TLS connections to the registry.
 
+ This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of Kommander.
```
Comment (cmastr) [inline pages/dkp/kommander/1.4/install-airgapped/index.md:99]: hey @mwrightd2 these messages need to match what's in this folder verbatim. 

https://drive.google.com/drive/u/0/folders/1eb6tnAx6Il9YvyVKZ2Oz0Scubhuwis7P

**Review by cmastr — DISMISSED** [review]
Need to update some text. 

https://drive.google.com/drive/u/0/folders/1eb6tnAx6Il9YvyVKZ2Oz0Scubhuwis7P

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 2]
```diff
@@ -110,3 +108,5 @@ To configure the Gitea server, follow these steps:
     ```
 
 1. After the newly created `GitRepository` on the management cluster reconciles, any corresponding `App`s are loaded by Kommander controller.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of DKP/Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of DKP/Kommander.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:112]: THought we were supposed to use Kevin's Kommander.txt files from google drive for these?
This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the GNU Affero General Public License 3.0. The complete source code for the versions of MinIO packaged with DKP/Kommander/Konvoy 2.1.1 are available at these URLs: 
https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z
https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z

File: pages/dkp/kommander/2.1/download/index.md [hunk 3]
```diff
@@ -39,3 +37,5 @@ dkp and kommander binaries placed in the local directory, to run:
 ```
 
 You will now see two binaries: `dkp` and `kommander` in your working directory. Follow the [Kommander installation instructions](../install/networked) using these binaries, and then [add your license](../licensing/add/) to Kommander. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of DKP/Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of DKP/Kommander.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.1/download/index.md:41]:
```suggestion
This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the GNU Affero General Public License 3.0. The complete source code for the versions of MinIO packaged with DKP/Kommander/Konvoy 2.1.1 are available at these URLs: 
https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z
https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z
```

File: pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md [hunk 4]
```diff
@@ -179,6 +176,8 @@ Before running the following command, ensure that your `dkp` configuration **ref
 
 You can always go back to the [DKP Upgrade overview][dkp_upgrade], to review the next steps depending on your environment and license type.
 
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of DKP/Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of DKP/Kommander.
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md:179]: This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the GNU Affero General Public License 3.0. The complete source code for the versions of MinIO packaged with DKP/Kommander/Konvoy 2.2.0 are available at these URLs: 
https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z
https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z

File: pages/dkp/kommander/2.1/download/index.md (hunk 3 above)
Comment (mmcelroy91) [reply pages/dkp/kommander/2.1/download/index.md:41]: and so forth for 2.1 then switch to the 2.2 for all 2.2 sections

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kommander/1.4/install-airgapped/index.md (hunk 1 above)
Comment (cneth) [inline pages/dkp/kommander/1.4/install-airgapped/index.md:99]: Isn't it really the 'konvoy_air_gapped.tar.gz' that contains the minio?

Comment (mwrightd2) [reply pages/dkp/kommander/1.4/install-airgapped/index.md:99]: I agree, but yeah this is just coming straight from the law horses mouth.

File: pages/dkp/kommander/2.1/install/air-gapped/index.md [hunk 5]
```diff
@@ -127,9 +127,7 @@ export VERSION=v2.1.1
     wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O kommander-image-bundle.tar
     ```
 
-    <p class="message--note"><strong>NOTE: </strong>This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License 3.0</a>. Complete source code for MinIO is available <a href="https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z">here</a> and <a href="https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z">here</a>.</p>
-
-1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Unpack the bundle and the nested bundle within in a location where you can load and push the images to your private Docker registry.
```
Comment (cneth) [inline pages/dkp/kommander/2.1/install/air-gapped/index.md:130]: This needs to be reviewed by whomever is packaging the new files.      If I understand correctly, you should NOT unpack twice but just once, the script below still takes a .tar file.

Also, I think this should be more explicit.  I think we should give the exact steps the user should execute here so there is no confusion; 'unbundle' is not specific enough.

File: pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md [hunk 6]
```diff
@@ -177,7 +175,7 @@ See [Kommander Load Balancing][kommander-load-balancing] for more information.
     wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-image-bundle-v2.2.0.tar.gz" -O catalog-applications-image-bundle.tar.gz
     ```
 
-1.  Place the bundles in a location where you can load and push the images to your private Docker registry.
+1.  Unpack the bundle and the nested bundle within in a location where you can load and push the images to your private Docker registry.
```
Comment (cneth) [inline pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md:178]: As above, please confirm with eng exactly what this step should say.

**Review by cneth — CHANGES_REQUESTED** [review]
Other than the 'unbundling' instructions, this looks ok to me.   But we need eng to confirm exactly what the steps are before committing.

File: pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md (hunk 6 above)
Comment (dkoshkin) [inline pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md:178]:
```suggestion
1.  See the `NOTICES.txt` file for the 3rd party software attributions and place `kommander-image-bundle-v2.2.0.tar.gz` and `dkp-catalog-applications-image-bundle-v2.2.0.tar.gz` bundles within in a location where you can load and push the images to your private Docker registry.
```

Comment (dkoshkin) [reply pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md:178]: Change
```
wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-image-bundle-v2.2.0.tar.gz" -O kommander-image-bundle.tar.gz
```
to
```
wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-image-bundle-v2.2.0.tar.gz" | tar -xvf -
```

and     
```
wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-image-bundle-v2.2.0.tar.gz" -O catalog-applications-image-bundle.tar.gz
```
to
```
wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-image-bundle-v2.2.0.tar.gz" | tar -xvf -
```

**Review by dkoshkin — APPROVED** [review]

**Review by cneth — APPROVED** [review]

**Pull request merged**